src/libraries/Common/src/Interop/Android/Interop.Logcat.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Logcat
+    {
+        [DllImport(Libraries.Liblog)]
```

Comment:
I don't like p/invoking directly into liblog. It feels much better to do it like we do for iOS and go into pal_log.c.

Reply:
@akoeplinger  why is it better? it's extra code and Interop* stuff tends to do direct pinvokes if it's possible, e.g. all these https://github.com/dotnet/runtime/blob/master/src/libraries/Common/src/Interop/OSX/Interop.Libraries.cs#L9-L11
also it will increase `libSystem.Native.so` dynamic lib size as I will have to link some android lib

Reply:
Hm yeah after thinking about it it might be fine, I got thrown off by 1) thinking the `__` are due to C++ name mangling but it's actually documented that way so should be fine from a stability point and 2) it's different to how we do it in mono/mono but again that should be fine.